rosdep/python.yaml

```diff
@@ -5998,6 +5998,10 @@ python3-pykdl:
     '*': [python3-pykdl]
     'bionic': null
     'xenial': null
+python3-pymodbus:
+  debian: [python3-pymodbus]
+  fedora: [python3-pymodbus]
+  ubuntu: [python3-pymodbus]
```

Comment:
There's a similar situation for Ubuntu; the key is not available in Xenial, so a null key there and `python3-pymodbus` everywhere else.

Reply:
My bad, addressed in https://github.com/ros/rosdistro/pull/26056/commits/85aee79a8a6de248b97f0596cfc9a2324b4d1025